website/pages/en/developing/supported-networks.mdx

```diff
@@ -4,35 +4,37 @@ title: Supported Networks
 
 import { getSupportedNetworks } from '@/src/getSupportedNetworks'
 
-## The Graph's Decentralized Network
-
-The following networks are supported on The Graph's Decentralized Network:
-
-<ListUnordered>
-  {getSupportedNetworks('studio').map((network) => (
-    <li key={network.cliName}>
-      <CodeInline>{network.cliName}</CodeInline>
-      {network.cliName === 'mainnet' ? ` (${network.name})` : null}
-      {network.beta ? '*' : null}
-    </li>
+Here is a table showing the networks that The Graph supports, whether it is on the hosted service, Subgraph Studio, or the decentralized network:
+
+<Table>
+  <tr>
+    <th>Network</th>
+    <th>CLI Name</th>
+    <th align="center">Chain ID</th>
+    <th align="center">Hosted Service</th>
+    <th align="center">Subgraph Studio</th>
+    <th align="center">Decentralized Network</th>
+  </tr>
+  {getSupportedNetworks().map((network) => (
+    <tr key={network.cliName}>
+      <td>
+        {network.name}
+        {network.isBeta ? '*' : null}
```

Comment:
I think we don't mention what the asterisks mean anywhere? I think this is principally relevant on the network, so maybe the asterisks should just show there (for chains which are on the decentralised network, next to the check mark?)